server/odc-migrate/src/main/resources/migrate/common/V_4_3_0_8__add_schedule_change_log.sql

```diff
@@ -14,6 +14,6 @@ CREATE TABLE IF NOT EXISTS `schedule_changelog` (
 CREATE TABLE IF NOT EXISTS `schedule_latest_task_mapping` (
 `id` bigint(20) NOT NULL AUTO_INCREMENT COMMENT 'The primary key identifier for this link record',
 `schedule_id` bigint NOT NULL COMMENT 'Identifier of the schedule, refer to schedule_schedule.id',
-`latest_schedule_task_id` bigint NULL COMMENT 'Identifier of the latest task within this schedule, can be NULL if no task has been run yet'
-) COMMENT='This table maintains the link to the latest task for each schedule for quick reference';
-
+`latest_schedule_task_id` bigint NULL COMMENT 'Identifier of the latest task within this schedule, can be NULL if no task has been run yet',
```

Comment:
we may add a new script to add this unique key~

Reply:
The SIT environment is not deployed yet, so it is simpler.

Reply:
this pr will be merged into dev/4.3.x finally and the 4.3.0 has already been released